witness/src/Dscp/Witness/Web/Server.hs

```diff
@@ -34,8 +35,11 @@ convertWitnessHandler
     -> Handler a
 convertWitnessHandler (UnliftIO unliftIO) handler =
     liftIO (unliftIO handler)
-        `catch` (throwError . witnessToServantErr)
-        `catchAny` (throwError . witnessToServantErr . InternalError . show)
+        `catch` throwServant
+        `catch` (throwServant . ServiceUnavailable . show @Text @RelayException)
```

Comment:
This way we would need to list all possible exceptions at every `catch` site (at client for instance, if it will be needed).
Better add `ServiceUnavailable <$> fromException e` here https://git.io/fAudK instead